backend/games/wordGuesser/ClassicWordle.py

```diff
@@ -0,0 +1,33 @@
+from typing import List
```

Comment:
Can you make the folder name `word_guesser` instead of `wordGuesser`? Our Python codebase uses snake_case for module/function names/variables and camelCase for class names. `ClassicWordle` is fine for the class name!

It would also be more consistent with the codebase as a whole if your functions and variables were also snake_case, but it's not the most important thing.